bbl-viewer.appcache

```diff
@@ -0,0 +1,116 @@
+CACHE MANIFEST
+# 2018-08-21:v1
+
+
+CACHE:
+index.html
```

Comment:
The process that this app is built with gulp is that all the required sources are copied, and the required node modules installed, into the `dist/` directory, and then packing happens in there. I think this should use the same process, and produce a zip file as the output.

Also, n.b. that this currently misses the node modules, and some functionality will not work because of that.

Reply:
Yep, that's true (node modules). I did not even bother yet to build them and expected functionality relying on them to fail - I was thinking of movie generation at that time but I later realized that there's more than that. The part which interested me on the field (eg. looking at the graphs and fft) was not impacted by that and this change made it possible to use the app without network connection.

You're absolutely right, the same process generating that zip file should generate the index as only at that point it's really sure what goes into the final distribution. Myself, I was lazy, I just dropped the git repository into my Apache and ignored node.

Reply:
> @aguindehi: Are you still working on this?

@mikeller Not actively. I never worked with Gulp so I would have to dig into that first. And I don't know Betaflight's build system yet. There's probably someone doing that better than myself to generate that list of files¸ someone who knows where to look and where to hook it in?

Reply:
Looking at gulpfile.js it seems that one would need to add a .pipe(array_to_manifest(distSources)) to function dist() and implement a array_to_manifest() function creating the manifest file in the process.